### ISSUE TITLE
[Arm64/Ubuntu] JitStressRegs=0x800 mostly fails

### ISSUE THREAD

**Issue by sdmaclea** [dotnet/coreclr#16476]
When setting `JitStressRegs=0x800` `LSRA_SPILL_ALWAYS`, there are about 9K fails.

@CarolEidt Is this a JitStressRegs mode that should work?

**Comment by CarolEidt** [comment]
No, I'm afraid to say that this mode has never fully worked. There is work to be done to ensure that it doesn't spill in places that it cannot. There should already have been an issue opened for this (it's an issue on all targets).

**Comment by CarolEidt** [comment]
Thanks for opening an issue, BTW
